### PR TITLE
Removed IViewRenderer

### PR DESCRIPTION
**Pull request by thecodejunkie**
### Prerequisites

- [X] I have written a descriptive pull-request title
- [X] I have verified that there are no overlapping [pull-requests](https://github.com/NancyFx/Nancy/pulls) open
- [X] I have verified that I am following the Nancy [code style guidelines](https://github.com/NancyFx/Nancy/blob/45238076ad0b7f6ecabd6bae8469e30458d02efe/CONTRIBUTING.md#style-guidelines)
- [X] I have provided test coverage for my change (where applicable)

### Description
The `IViewRenderer` interface was no longer being used. Removed it, the implementation, the tests and the registration in the container

### CODE REVIEW

File: src/Nancy/Bootstrapper/NancyBootstrapperBase.cs [hunk 1]
```diff
@@ -608,11 +608,9 @@ protected virtual void Dispose(bool disposing)
         /// that don't form part of the core Nancy configuration
         /// </summary>
         /// <returns>Collection of TypeRegistration types</returns>
-        private IEnumerable<TypeRegistration> GetAdditionalTypes()
+        private static IEnumerable<TypeRegistration> GetAdditionalTypes()
```
Comment (khellang) [inline src/Nancy/Bootstrapper/NancyBootstrapperBase.cs]: Can we just remove this method?

**Comment by thecodejunkie** [comment]
@khellang done! 👍

**Comment by throwException** [comment]
I previously used IViewRenderer in my custom status page. What's the substitute to render a custom status page with SSVE?

**Comment by khellang** [comment]
Just inject `IViewFactory` and copy the code instead? It's like 2 lines 😉

**Pull request merged**